qiita_db/support_files/patches/14.sql

```diff
@@ -0,0 +1,2 @@
+-- we don't really need an SQL patch, but having nothing here breaks the build
+select 1+1;
```

Comment:
Fixing in my next commit, thanks for catching this major oversight on my
end.

On (Feb-11-15|13:10), Daniel McDonald wrote:
> > @@ -0,0 +1,2 @@
> > +-- we don't really need an SQL patch, but having nothing here breaks the build
> > +select 1+1;
> 
> Obviously this should be `SELECT 42`
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/829/files#r24533561